optaplanner-core/src/main/java/org/optaplanner/core/impl/score/DefaultScoreManager.java

```diff
@@ -38,10 +39,22 @@ public DefaultScoreManager(ScoreDirectorFactory<Solution_> scoreDirectorFactory)
 
     @Override
     public void updateScore(Solution_ solution) {
+        getScore(solution);
```

Comment:
ok, so getScore() doesn't just extract it, but calculates it?
We can't have 2 methods that do exactly the same, so I guess updateScore() needs to be deprecated?